### PR TITLE
new statement for alpha collectors and metrics in README

### PR DESCRIPTION
**Pull request by zouyee**
/cc @brancz  @andyxning 

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kube-state-metrics/294)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,14 +78,22 @@ those that could be used for actionable alerts. Please contribute PR's for
...
 >	* kube_pod_container_resource_limits_nvidia_gpu_devices
 >	* kube_node_status_capacity_nvidia_gpu_cards
 >	* kube_node_status_allocatable_nvidia_gpu_cards
 >
 >	are in alpha stage and will be deprecated when the kubernetes gpu support is final in 1.9 version.
+>
+> **collectors**
+> * v2alpha1/cronjob
+>
+> If users want to enable this feature when kubernetes version larger than 1.8,
```
Comment (brancz) [inline README.md]: 1.8 -> 1.7

File: README.md [hunk 2]
```diff
@@ -78,14 +78,22 @@ those that could be used for actionable alerts. Please contribute PR's for
...
 >
+> **metrics**
 >	* kube_pod_container_resource_requests_nvidia_gpu_devices
 >	* kube_pod_container_resource_limits_nvidia_gpu_devices
 >	* kube_node_status_capacity_nvidia_gpu_cards
 >	* kube_node_status_allocatable_nvidia_gpu_cards
 >
 >	are in alpha stage and will be deprecated when the kubernetes gpu support is final in 1.9 version.
+>
+> **collectors**
```
Comment (brancz) [inline README.md]: In addition I think we can add a catch all statement for all collectors and metrics based on alpha features, we cannot provide any stability guarantees.

Comment (zouyee) [reply README.md]: would we split `collector and metrics` statement info into each collector README or create a new file?

Comment (brancz) [reply README.md]: Just a statement above here that says something like: "Any collectors and metrics based on alpha Kubernetes APIs are excluded from *any* stability guarantee. They may change at any given release."

Comment (zouyee) [reply README.md]: got it, thank you @brancz

**Comment by brancz** [comment]
lgtm :+1: thanks!

**Pull request merged**